src/gui/wizard/owncloudwizard.cpp

```diff
@@ -161,13 +152,17 @@ void OwncloudWizard::successfulStep()
         break;
 
     case WizardCommon::Page_ServerSetup:
-    case WizardCommon::Page_Result:
         qCWarning(lcWizard, "Should not happen at this stage.");
         break;
     }
 
     ownCloudGui::raiseDialog(this);
-    next();
+    if (nextId() == -1) {
```

Comment:
Perhaps

```suggestion
    if (nextId() < -1) {
```

Otherwise, it might make sense to just use `ssize_t` instead of `int`.

Reply:
<= I assume :D

Reply:
But only -1 is a valid page if you consider the doc